modules/features2d/misc/java/test/SIMPLEBLOBFeatureDetectorTest.java

```diff
@@ -1,11 +1,58 @@
-package org.opencv.test.features2d;
```

Comment:
Any reason why the `package` is removed? The declaration exists in other test cases as well.